### PR TITLE
:package: v2021-02-27

### PR DESCRIPTION
**Pull request by kba**
@stweil Can/should we update tesseract to master?

### CODE REVIEW

**Comment by stweil** [comment]
Yes, I think so. See also https://github.com/tesseract-ocr/tesseract/pull/3141#issuecomment-785896647.

**Comment by kba** [comment]
> Yes, I think so. See also [tesseract-ocr/tesseract#3141 (comment)](https://github.com/tesseract-ocr/tesseract/pull/3141#issuecomment-785896647).

Building tesserocr [failed](https://app.circleci.com/pipelines/github/OCR-D/ocrd_all/501/workflows/6ae977f3-2de7-49fa-9bea-21cf04d226b1/jobs/495) with

```
tesserocr.cpp:681:10: fatal error: tesseract/genericvector.h: No such file or directory
   #include "tesseract/genericvector.h"
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
```

@stweil Since https://github.com/sirfz/tesserocr/pull/242 is merged but no new release yet available, we'll have to change to using tesserocr GH master instead of PyPI for the time being, correct?

**Comment by stweil** [comment]
As far as I know we use `tesserocr` submodule (not PyPI), but the submodule needs of course an update if it does not include that commit.

**Comment by stweil** [comment]
@kba, I just fixed a major issue for Tesseract training. Maybe you want an update to latest git master to get that fix, too.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,413 @@
...
+  > * 🚧 dinglehopper: Try out Drone CI
+  > * 🚧 dinglehopper: Try out Drone CI
+  > * 🚧 dinglehopper: Try out Drone CI
+  > * 🚧 dinglehopper: Try out Drone CI
+  > * 🚧 Replace Travis with CircleCI
+  > * 🚧 Replace Travis with CircleCI
+  > * 🚧 Replace Travis with CircleCI
+  > * 🚧 Replace Travis with CircleCI
+  > * 🚧 Replace Travis with CircleCI
+  > * 🚧 Replace Travis with CircleCI
```
Comment (bertsky) [inline CHANGELOG.md:41]: Is this repetition of log messages something we could suppress in our script for the aggregated changelog?

**Review by bertsky — CHANGES_REQUESTED** [review]
Please update ocrd_segment once more. (Contains a fallback in case there's no ReadingOrder.)

File: CHANGELOG.md (hunk 1 above)
Comment (kba) [reply CHANGELOG.md:41]: We could pipe the output through `uniq`.

Comment (kba) [reply CHANGELOG.md]: done 6229c7d

**Pull request merged**